### PR TITLE
feat(tokens): Update color tokens

### LINKED ISSUE #28831: Feature: Token addition requests
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
Fluent v9
```

### Are you reporting Accessibility issue?

None

### Reproduction

React Components / v9

## Brand background 3 Static
image

## Brand background 4 Static
image

------
## colorPaletteRedBackground3Hover
image

## colorPaletteRedBackground3Pressed
image

cc: @miroslavstastny 

### Logs

_No response_

### Requested priority

High

### Products/sites affected

Teams

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### LINKED ISSUE #29330: [Bug]: Update Dark theme Danger Foreground 1 from Cranberry 30 to Cranberry 40 to address accessibility issues.
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/theme-colors--page
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://aka.ms/fluentui-storybook

### Bug Description

Currently, dark theme Danger Foreground 1 does not pass the minimum 4.5:1 contrast requirement against Fluent 2 surfaces.

CleanShot 2023-09-28 at 09 08 07@2x

We will need to increase the contrast to meet the minimum contrast requirement and ideally have similar perceived contrast values to the other categories (e.g. Success FG1, Warning FG1)

CleanShot 2023-09-28 at 09 17 16@2x

## To-do
- Change dark theme `Danger Foreground 3` from `Cranberry Tint 30 (#DC626D)` to `Cranberry Tint 40 (#EEACB2)`

### Logs

```shell
https://aka.ms/fluentui-storybook
```

### Requested priority

High

### Products/sites affected

https://aka.ms/fluentui-storybook

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### LINKED ISSUE #30037: [Feature]: Neutral Card Background 1 - Colour token request
### Library

~~React Northstar / v0 (@fluentui/react-northstar)~~

v9

### Describe the feature that you would like added

Request to add this colour token in the V9 library. This is for the cards design that we recently implemented in Teams. 

Card combanitions

### Have you discussed this feature with our team

Yes

### Additional context

_No response_

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Priority

Normal

### PR DESCRIPTION
**Pull request by miroslavstastny**
## New Behavior

This PR adds the following color tokens:

| token | Light theme | Dark theme | High contrast |
|---|---|---|---|
| colorBrandBackground3Static | brand.60 | brand.60 | hcCanvas |
| colorBrandBackground4Static | brand.40 | brand.40 | hcCanvas |
| colorNeutralCardBackground | grey[98] | grey[20] | hcCanvas |
| colorNeutralCardBackgroundHover | white | grey[24] | hcHighlight |
| colorNeutralCardBackgroundPressed | grey[96] | grey[18] | hcHighlight |
| colorNeutralCardBackgroundSelected | grey[92] | grey[22] | hcHighlight |
| colorNeutralCardBackgroundDisabled | grey[94] | grey[8] | hcCanvas |
| colorStatusDangerBackground3Hover | cranberry.shade10 | cranberry.shade10 | hcHighlight |
| colorStatusDangerBackground3Pressed | cranberry.shade20 | cranberry.shade20 | hcHighlight |

image

It also updates `colorStatusDangerForeground3` **in dark theme** from cranberry.tint30 (#DC626D) to cranberry.tint40 (#EEACB2) to address accessibility issue:

image

## Related Issue(s)

- Fixes #28831  
- Fixes #29330
- Fixes #30037

### CODE REVIEW

File: packages/tokens/src/types.ts [hunk 1]
```diff
@@ -123,6 +123,8 @@ export type ColorTokens = {
   colorBrandBackground2: string;
   colorBrandBackground2Hover: string;
   colorBrandBackground2Pressed: string;
+  colorBrandBackground3Static: string;
```
Comment (miroslavstastny) [inline packages/tokens/src/types.ts:126]: To confirm with @daisygeng - there is nothing like `colorBrandBackground2Static`.

File: packages/tokens/src/global/colors.ts [hunk 2]
```diff
@@ -291,12 +291,12 @@ export const marigold: ColorVariants = {
 };
 
 export const yellow: ColorVariants = {
-  primary: '#fde300',
-  shade10: '#e4cc00',
-  shade20: '#c0ad00',
-  shade30: '#817400',
-  shade40: '#4c4400',
   shade50: '#282400',
```
Comment (miroslavstastny) [inline packages/tokens/src/global/colors.ts:294]: There is no change to color tokens here. Just reordered to match token pipeline output.

File: packages/tokens/src/types.ts [hunk 3]
```diff
@@ -210,6 +212,8 @@ export type ColorPaletteRed =
   | 'colorPaletteRedBackground1'
   | 'colorPaletteRedBackground2'
   | 'colorPaletteRedBackground3'
+  | 'colorPaletteRedBackground3Hover'
```
Comment (miroslavstastny) [inline packages/tokens/src/types.ts:215]: To confirm with @daisygeng - are these `red` or semantic colors (`danger`)?

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/Avatar-mount_base.html">628</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/Avatar-mount.html">632</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/Button-mount_base.html">301</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/Button-mount.html">298</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/Field-mount_base.html">1127</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/Field-mount.html">1166</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/FluentProvider-mount_base.html">702</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/FluentProvider-mount.html">713</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/FluentProviderWithTheme-mount_base.html">80</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/FluentProviderWithTheme-mount.html">85</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">62</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">63</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">67</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">76</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/MakeStyles-mount_base.html">862</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/MakeStyles-mount.html">858</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/Persona-mount_base.html">1739</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/Persona-mount.html">1710</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/SpinButton-mount_base.html">1393</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30412/perf-test-react-components/SpinButton-mount.html">1409</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                                                |     Baseline (minified/GZIP) |                           PR |                                                                                                                                                                                                  Change |
| :--------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: | ---------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr> | `69.901 kB`<br />`20.261 kB` | `70.071 kB`<br />`20.285 kB` | `170 B` <br />`24 B`  |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>               | `42.388 kB`<br />`14.103 kB` |  `42.558 kB`<br />`14.13 kB` | `170 B` <br />`27 B`  |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                                                     |  `34.616 kB`<br />`7.317 kB` |  `35.149 kB`<br />`7.375 kB` | `533 B` <br />`58 B`  |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                                                   |  `19.287 kB`<br />`5.498 kB` |  `19.453 kB`<br />`5.522 kB` | `166 B` <br />`24 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContext.fixture.js'>createContext</abbr>                                                                    |          `510 B`<br />`328 B` |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContextSelector.fixture.js'>createContextSelector</abbr>                                                    |          `537 B`<br />`339 B` |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |  `93.962 kB`<br />`28.706 kB` |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                                       |  `83.737 kB`<br />`23.474 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `50.175 kB`<br />`15.944 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                                          |   `19.704 kB`<br />`7.796 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                                  |  `64.831 kB`<br />`20.274 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |   `26.905 kB`<br />`8.729 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |   `27.806 kB`<br />`9.024 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                                       |   `25.311 kB`<br />`9.306 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |  `39.513 kB`<br />`11.169 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |  `46.874 kB`<br />`12.661 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |  `44.292 kB`<br />`12.542 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |  `52.306 kB`<br />`14.134 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |  `56.559 kB`<br />`13.068 kB` |
| <samp>react-calendar-compat</samp> <br /> <abbr title='bundle-size/Calendar.fixture.js'>Calendar Compat</abbr>                                                                |  `142.28 kB`<br />`37.196 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `94.653 kB`<br />`27.242 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `89.477 kB`<br />`25.731 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |   `13.064 kB`<br />`5.385 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |   `15.307 kB`<br />`6.143 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |   `14.015 kB`<br />`5.752 kB` |
| <samp>react-checkbox</samp> <br /> <abbr title='bundle-size/Checkbox.fixture.js'>Checkbox</abbr>                                                                              |  `34.863 kB`<br />`11.881 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                                 |  `97.454 kB`<br />`31.473 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                                 |  `98.675 kB`<br />`31.468 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `210.791 kB`<br />`60.132 kB` |
| <samp>react-datepicker-compat</samp> <br /> <abbr title='bundle-size/DatePicker.fixture.js'>DatePicker Compat</abbr>                                                          | `214.223 kB`<br />`60.082 kB` |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr>                                                    |  `95.522 kB`<br />`28.482 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |    `20.82 kB`<br />`7.763 kB` |
| <samp>react-field</samp> <br /> <abbr title='bundle-size/Field.fixture.js'>Field</abbr>                                                                                       |    `22.544 kB`<br />`8.61 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |   `15.736 kB`<br />`6.228 kB` |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoButton.fixture.js'>InfoButton</abbr>                                                                        | `133.068 kB`<br />`41.876 kB` |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoLabel.fixture.js'>InfoLabel</abbr>                                                                          | `136.873 kB`<br />`43.087 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |   `26.887 kB`<br />`9.118 kB` |
| <samp>react-jsx-runtime</samp> <br /> <abbr title='bundle-size/Classic.fixture.js'>Classic Pragma</abbr>                                                                      |       `1.057 kB`<br />`530 B` |
| <samp>react-jsx-runtime</samp> <br /> <abbr title='bundle-size/DevRuntime.fixture.js'>JSX Dev Runtime</abbr>                                                                  |    `3.783 kB`<br />`1.648 kB` |
| <samp>react-jsx-runtime</samp> <br /> <abbr title='bundle-size/Runtime.fixture.js'>JSX Runtime</abbr>                                                                         |    `4.377 kB`<br />`1.881 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                                       |   `14.266 kB`<br />`5.828 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |   `17.082 kB`<br />`6.911 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          |   `143.091 kB`<br />`43.8 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `145.777 kB`<br />`44.307 kB` |
| <samp>react-message-bar</samp> <br /> <abbr title='bundle-size/Default.fixture.js'>MessageBar (all components)</abbr>                                                         |   `25.181 kB`<br />`9.039 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                                                               |   `12.862 kB`<br />`4.827 kB` |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                                                                                 |  `57.066 kB`<br />`17.821 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 |  `121.262 kB`<br />`38.27 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |    `12.788 kB`<br />`4.67 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `7.099 kB`<br />`2.385 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                                                                    |   `26.278 kB`<br />`9.496 kB` |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/ProgressBar.fixture.js'>ProgressBar</abbr>                                                                        |   `17.428 kB`<br />`6.898 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |   `22.928 kB`<br />`8.472 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                                                                                       |  `32.159 kB`<br />`10.064 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                                                                             |   `15.345 kB`<br />`6.263 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |  `28.609 kB`<br />`10.205 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |   `39.15 kB`<br />`12.769 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                                                                        |  `36.774 kB`<br />`11.788 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                                 |   `23.517 kB`<br />`8.512 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |  `34.347 kB`<br />`11.009 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                                                                                 |  `157.48 kB`<br />`43.924 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                                            |  `44.493 kB`<br />`13.921 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                                 | `130.438 kB`<br />`35.166 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                                         |  `75.482 kB`<br />`20.352 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                                   |   `74.148 kB`<br />`19.96 kB` |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/InteractionTag.fixture.js'>InteractionTag</abbr>                                                                      |   `15.259 kB`<br />`6.058 kB` |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/tag.fixture.js'>Tag</abbr>                                                                                            |   `29.982 kB`<br />`9.439 kB` |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/TagGroup.fixture.js'>TagGroup</abbr>                                                                                  |   `74.404 kB`<br />`22.31 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                                        |   `16.705 kB`<br />`6.569 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                                      |   `19.878 kB`<br />`6.896 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                                                              |  `30.947 kB`<br />`10.476 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>                                                        |            `69 B`<br />`89 B` |
| <samp>react-timepicker-compat</samp> <br /> <abbr title='bundle-size/TimePicker.fixture.js'>TimePicker</abbr>                                                                 |  `99.288 kB`<br />`32.906 kB` |
| <samp>react-toast</samp> <br /> <abbr title='bundle-size/Toast.fixture.js'>Toast (including Toaster)</abbr>                                                                   |  `93.274 kB`<br />`28.044 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `53.631 kB`<br />`18.944 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                                                                       |          `180 B`<br />`160 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/2c3fdb3e90f95f7476d02a4b3b253c5f04bc85cf'>2c3fdb3e90f95f7476d02a4b3b253c5f04bc85cf</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/public-docsite-v9/src/Concepts/Migration/FromV8/ThemeColors/colorMap.ts [hunk 4]
```diff
@@ -208,6 +208,8 @@ export const alias = {
   colorBrandBackground2: 'brand[160]',
```
[the remark this answers is not in the document]
Comment (miroslavstastny) [reply apps/public-docsite-v9/src/Concepts/Migration/FromV8/ThemeColors/colorMap.ts:208]: Change of error icon color in dark theme:
Baseline:
image

Candidate:
image

[the remark this answers is not in the document]
Comment (miroslavstastny) [reply apps/public-docsite-v9/src/Concepts/Migration/FromV8/ThemeColors/colorMap.ts:208]: @redongreen - any advice on this one?
This is a consequence of #29330

[the remark this answers is not in the document]
Comment (redongreen) [reply apps/public-docsite-v9/src/Concepts/Migration/FromV8/ThemeColors/colorMap.ts:208]: Hi @miroslavstastny, thanks for tagging, you pointed out a mistake in the original ticket. 

This change is not intended to change the Danger Foreground color used on Danger Tint background. 

Danger Foreground 1 - This is the foreground used and optimized to work on Danger Background tint color. 
**Danger Foreground 3** - This is the foreground optimized to be used directly against the F2 web neutral background colors. 

In dark theme, Danger foreground 1 should remain as Cranberry Tint 30 (#DC626D). 

Therefore,

### Change 
Change dark theme `Danger Foreground 3` **from** `Cranberry Tint 30 (#DC626D)` **to** `Cranberry Tint 40 (#EEACB2)`

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/tokens/src/types.ts (hunk 1 above)
Comment (miroslavstastny) [reply packages/tokens/src/types.ts:126]: Discussed async, this is intentional to match `colorBrandBackground3` and `colorBrandBackground4`

File: packages/tokens/src/types.ts (hunk 3 above)
Comment (miroslavstastny) [reply packages/tokens/src/types.ts]: These are now `danger` tokens.

**Review by layershifter — APPROVED** [review]

**Review by GeoffCoxMSFT — APPROVED** [review]

**Pull request merged**